### DISCUSSION TITLE
I can't login as a super admin.

### DISCUSSION THREAD

**Discussion by Bitsafeti** [chatwoot/chatwoot#5166] [Q&A]
Hello my friends, I've tried in every way, but I can't access as a super admin. I've already created the user from the command line and I've used the user created in the setup but none of them work.

**Comment by vishnu-narayanan** [comment]
Could you please share the error you are getting if any? Also any error messages on the browser console?

**Comment by aei-gt** [comment]
i have same problem, install from script https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm but never request a user/pass to install (just the root) and after install can't login because just show normal login, try to use url to admin but redirect. Any know the problem, or how create a super admin from cwctl ? 
![image](https://github.com/chatwoot/chatwoot/assets/111803510/57139957-a30f-4f2f-8d85-8d7a8abce891)

**Comment by alimoosavi** [comment]
> Hello my friends, I've tried in every way, but I can't access as a super admin. I've already created the user from the command line and I've used the user created in the setup but none of them work.

What was the command that you used to create your super admin user?
